### DISCUSSION TITLE
How do you log a context object?

### DISCUSSION THREAD

**Discussion by jasonkuhrt** [community/community#25630] [Actions]
<p>I would like to log context objects to debug my workflow. How can I do that?</p>
<pre><code class="lang-auto">jobs:
  foo:
    steps:
      # How do I see the contents of that object?
      # The follow just prints "object"
      - run: echo "${{ github }}"
</code></pre>

**Comment by airtower-luna** [comment, marked as answer]
<p>Try using the <a href="https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#tojson"><code>toJSON()</code> function</a> on the object. When constructing <code>run</code> steps keep in mind that the shell treats some of the characters that occur in JSON specially (in particular quotes). :wink:</p>
